src/main/java/kg/jarkyn/cobspecserver/responders/OptionsResponder.java

```diff
@@ -0,0 +1,20 @@
+package kg.jarkyn.cobspecserver.responders;
+
+import kg.jarkyn.cobspecserver.data.Request;
+import kg.jarkyn.cobspecserver.data.Response;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class OptionsResponder extends Responder {
+    private List<String> methods = Arrays.asList("GET", "HEAD", "POST", "OPTIONS", "PUT");
```

Comment:
I wanted to have this defined separately, however for the time being I did the simplest thing.
In fact my first implementation was able to return OPTIONS for any recognised path